src/main/webapp/i18n/de/global.json

```diff
@@ -67,6 +67,7 @@
                 "shortAnswerSubmittedText": "Short Answer Submitted Text",
                 "shortAnswerMapping": "Short Answer Mapping",
                 "programmingExerciseTestCase": "Programming Exercise Test Case",
+                "exerciseHint": "Exercise Hint",
```

Comment:
should this not be German?

Reply:
Strange, seems like Jhipster always just uses the english name here for the entities. I translated the ExerciseHint to "Aufgabenhinweis" here.